easybuild/easyconfigs/p/Python/Python-3.9.5-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,441 @@
+name = 'Python'
+version = '3.9.5'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['e0fbd5b6e1ee242524430dee3c91baf4cbbaba4a72dd1674b90fda87b713c7ab']
+
+builddependencies = [
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('binutils', '2.36.1'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.35.4'),
+    ('XZ', '5.2.5'),
+    ('GMP', '6.2.1'),  # required for pycrypto
+    ('libffi', '3.3'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # Python 3.7 requires OpenSSL >= 1.0.2, uncomment line below if OS version is too old (e.g. CentOS 6)
+    # ('OpenSSL', '1.1.1k'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
+
+# symlink 'pip' command to 'pip3' that is included with Python installation
+# required so we can update pip to version included in extensions, using pip
+installopts = " && ln -s %(installdir)s/bin/pip3 %(installdir)s/bin/pip"
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 10 May 2021
+exts_list = [
```

Comment:
Order must be: setuptools, wheel, pip due to dependence, see https://github.com/easybuilders/easybuild-easyconfigs/pull/12650

Reply:
If I use this order I am getting `error: invalid command 'bdist_wheel'` for `setuptools`. I assume the dependencies of `setuptools` changed. According to the documentation of `setuptools` v56.2.0 `wheel` is needed.

For me the order `wheel, setuptools, pip` worked.

Reply:
Wow, great. So wheel needs setuptools and setuptools needs wheel :D
But glad you figured this out, this works